src/test/mediapipe/config_mp_passthrough.json

```diff
@@ -0,0 +1,9 @@
+{
+    "model_config_list": [],
```

Comment:
This test will probably fail when I merge my commit.
My commit fixes the issue with existence of `subconfig.json` in tests which adds dummy model.
Here you want to use dummy model but do not load dummy model in line 2.
Keep it in mind before merging PR

Reply:
Here test uses subconfig.json